commit a279a1453094d0abfc847aa790bc82addc14cc62
Author: Remi Gacogne <remi.gacogne@powerdns.com>
Date:   Thu Oct 16 15:11:56 2025 +0200

    dnsdist: Fix build error when only protobuf is enabled
    
    The build fails with:
    ```
    ../dnsdist-protobuf.cc: In member function ‘void DNSDistProtoBufMessage::serialize(std::string&) const’:
    ../dnsdist-protobuf.cc:205:7: error: ‘vinfolog’ was not declared in this scope
      205 |       vinfolog("Error while parsing the RRs from a response packet to add them to the protobuf message: %s", exp.what());
          |       ^~~~~~~~
    ```
    because of a missing header.
    
    Signed-off-by: Remi Gacogne <remi.gacogne@powerdns.com>

diff --git a/dnsdistdist/dnsdist-protobuf.cc b/pdns/dnsdistdist/dnsdist-protobuf.cc
index 73f4e43f54..2f984fc92b 100644
--- a/dnsdist-protobuf.cc
+++ b/dnsdist-protobuf.cc
@@ -27,6 +27,7 @@
 #include "base64.hh"
 #include "dnsdist.hh"
 #include "dnsdist-protobuf.hh"
+#include "dolog.hh"
 #include "protozero.hh"
 
 DNSDistProtoBufMessage::DNSDistProtoBufMessage(const DNSQuestion& dnsquestion) :
